src/gui/helpers/nav_helper.rs

```diff
@@ -111,7 +113,7 @@ impl GuiHelper for NavHelper {
         egui::CollapsingHeader::new("Boat Coordinates NOT IMPLEMENTED")
```

Comment:
Presumably actually implemented now, if it works. Need to play the game for a few hours more to test it :P